src/app/core/services/util/util.service.ts

```diff
@@ -29,4 +34,77 @@ export class UtilService {
         });
     };
   }
+
+  async alertPopup(msg) {
+    return new Promise(async (resolve) => {
+      let texts:any;
+      this.translate.get([msg.header, msg.message,msg.cancel, msg.submit]).subscribe(text => {
+        texts = text;
+      })
+      const alert = await this.alert.create({
+        cssClass: 'my-custom-class',
+        header: texts[msg.header],
+        message: texts[msg.message],
+        buttons: [
+          {
+            text: texts[msg.cancel],
+            role: 'cancel',
+            cssClass: 'secondary',
+            handler: (blah) => {
+              resolve(false);
+            }
+          }, {
+            text: texts[msg.submit],
+            handler: () => {
+              resolve(true);
+            }
+          }
+        ]
+      });
+      await alert.present();
+    });
+  }
+
+  getActionSheetButtons(type) {
```

Comment:
do we need this function here

Reply:
Here am fetching action sheet buttons based on parameter